debug-on-kubernetes.md

```diff
@@ -13,6 +13,10 @@ One of the key features of VS Code Kubernetes Extension is its one-click debuggi
 ## 3. Extension Settings for debugging
    * `vs-kubernetes` - Parent for Kubernetes-related extension settings
       * `vs-kubernetes.autoCleanupOnDebugTerminate` - The flag to control whether to auto cleanup the created Deployment and associated Pod by the command "Kubernetes: Debug (Launch)". The cleanup action occurs when it failed to start debug session or debug session terminated. If not specified, the extension will prompt for whether to clean up or not. You might choose not to clean up if you wanted to view pod logs, etc.
+      * `vs-kubernetes.nodejs-autodetect-remote-root` - If true will try to automatically get the root location of the source code in the container. The detection attempts to get the CWD of the nodejs process.  

```

Comment:
I love that you've detailed the settings here - great attention to detail!  Thank you!